Neos.Flow/Classes/Command/CacheCommandController.php

```diff
@@ -418,6 +418,39 @@ public function setupAllCommand(bool $quiet = false)
         }
     }
 
+    /**
+     * Cache Garbage Collection
+     *
+     * Runs the Garbage Collection (collectGarbage) method on all registered caches.
+     *
+     * Though the method is defined in the BackendInterface, the implementation
+     * can differ and might not remove any data, depending on possibilities of
+     * the backend.
+     *
+     * @param string $cacheIdentifier If set, this command only applies to the given cache
+     * @return void
+     * @throws NoSuchCacheException
+     */
+    public function garbageCollectionCommand(string $cacheIdentifier = null) {
```

Comment:
Adjust to PSR-2 to mage Style checker happy. And please add `:void` as return type.